torchvision/models/segmentation/segmentation.py

```diff
@@ -103,8 +119,13 @@ def _segm_lraspp_mobilenetv3(backbone_name, num_classes, pretrained_backbone=Tru
     return model
 
 
-def fcn_resnet50(pretrained=False, progress=True,
-                 num_classes=21, aux_loss=None, **kwargs):
+def fcn_resnet50(
+    pretrained: bool = False,
+    progress: bool = True,
+    num_classes: int = 21,
+    aux_loss: Optional[bool] = None,
+    **kwargs: Any
+) -> nn.Module:
```

Comment:
```suggestion
) -> FCN:
```

We can keep your [original idea](https://github.com/pytorch/vision/pull/4227/commits/9dd0ce74cff662b26603321ba1e99e1665e6fc58#diff-ccff7af514d99eeb40416c8b9ec30f032d1a3f450aaa4057958ca39ab174452eL129) of returning public types on these public methods.